### DISCUSSION TITLE
Can't get faces clusters to show on new installation

### DISCUSSION THREAD

**Discussion by martinomh** [photoprism/photoprism#1712] [Q&A] [answered]
Hi to the community!

I've recently installed Photoprism on my Raspberry Pi 3B+, using Docker Compose... Everything works fine, but seems that I'm unable to get faces showing in the People section. Version running: 211018-e200f322-Linux-armv7l. Is it because there aren't enough resources?

I've already done "Reset People & Faces" with command [docker-compose exec photoprism photoprism faces reset -f] but to no avail.

I've currently configured 1 GB of swap memory on my RPi. Do I need to upgrade that? Eventually, how much space do you recommend for swap?

Tensorflow is currently disabled (auto-disabled during setup). Is it needed for faces clustering?

Thanks everybody for every possible tips & suggestions.

**Comment by lastzero** [comment, marked as answer]
You need 2 GB of physical memory and TensorFlow must be enabled for face detection to work.

**Reply by martinomh** [reply]
Aw, so it seems I'm in a bad luck since RPi B+ has only 1GB of phisical memory and a not enough powerful CPU... did I get this right?

Or would you reccommend trying to expand the swap memory and enable Tensorflow anyway?

**Reply by lastzero** [reply]
Add 4 GB of Swap and set PHOTOPRISM_UNSAFE to "true" if you want to try.
